gcloud-java-storage/src/main/java/com/google/gcloud/storage/BucketInfo.java

```diff
@@ -434,33 +361,57 @@ Builder selfLink(String selfLink) {
       return this;
     }
 
+    /**
+     * Sets whether blob versioning should be enabled or not for the bucket.
+     */
     public Builder versioningEnabled(Boolean enable) {
       this.versioningEnabled = firstNonNull(enable, Data.<Boolean>nullOf(Boolean.class));
       return this;
     }
 
+    /**
+     * Sets the bucket's website index page.
+     */
     public Builder indexPage(String indexPage) {
       this.indexPage = indexPage;
       return this;
     }
 
+    /**
+     * Sets the bucket's website page to return when a resource is not found.
+     */
     public Builder notFoundPage(String notFoundPage) {
       this.notFoundPage = notFoundPage;
       return this;
     }
 
+    /**
+     * Sets bucket's delete rules.
+     *
+     * @see <a href="https://cloud.google.com/storage/docs/lifecycle">Lifecycle Management</a>
+     */
     public Builder deleteRules(Iterable<? extends DeleteRule> rules) {
       this.deleteRules = rules != null ? ImmutableList.copyOf(rules) : null;
       return this;
     }
 
-    public Builder storageClass(StorageClass storageClass) {
-      this.storageClass = firstNonNull(storageClass, StorageClass.NULL_VALUE);
+    /**
+     * Sets the bucket's storage class. A list of supported values is available
+     * <a href="https://cloud.google.com/storage/docs/json_api/v1/buckets#resource">here</a>.
```

Comment:
Based on our comments the doc was updated to include the list of values in https://cloud.google.com/storage/docs/storage-classes (see "Name to use with APIs").